tips/tests/tips_tests.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+from django.utils import timezone
+from tips.models import Tip
+
+AUTHOR = 'RICK'
+DATE = timezone.now()
+CATEGORY = 'Clothing'
+TIP = 'I BOUGHT A TESLA IN 50$ DISCOUNT IN LEVINSKI MARKET'
```

Comment:
Try changing  the constant TIP to a different name, it might have caused the issues with the tests

Reply:
Tnak you It realy solved it

Reply:
Great work! you can mark this as resolved (it seems like i can't do it myself)